src/concrete/ml/sklearn/qnn.py

```diff
@@ -594,6 +590,7 @@ def dump_dict(self) -> Dict[str, Any]:
         metadata["history_"] = self.history_
         metadata["initialized_"] = self.initialized_
         metadata["virtual_params_"] = self.virtual_params_
+        metadata["classes_"] = self.classes_
```

Comment:
this was forgotten when adding serialization to QNNs (since it was not really needed), so I needed to add it here